Tests/BugsnagTests/PayloadTransformerTests.swift

```diff
@@ -37,6 +40,12 @@ class PayloadTransformerTests: XCTestCase {
         self.payload = try! self.payloadTransformer.payloadFor(
             message: "Test message",
             metadata: Node(["key": "value"]),
+            request: req,
+            severity: self.severity
```

Comment:
Consider setting the severity explicit here instead of referring to the `severity` variable. Worst case, somebody could change the value of the variable.